code/modules/mob/living/carbon/human/species_types/skeletons.dm

```diff
@@ -29,6 +29,9 @@
 		if(chem.volume > 10)
 			H.reagents.remove_reagent(chem.type, chem.volume - 10)
 			to_chat(H, "<span class='warning'>The excess milk is dripping off your bones!</span>")
+		for(var/i in H.all_wounds)

```

Comment:
you forgot to add this for bone golems too

Reply:
done